client/cli/ok.py

```diff
@@ -119,7 +119,7 @@ def main():
     # Instantiating assignment
     try:
         assign = assignment.load_config(args.config, args)
-    except (ex.LoadingException, ex.SerializeException) as e:
+    except ex.OkException as e:
```

Comment:
I'll change it to ex.LoadingException, since SerializeException is a subclass of LoadingException.